pkg/jobs/jobspb/jobs.proto

```diff
@@ -329,6 +329,9 @@ message RestoreDetails {
 
   // DebugPauseOn describes the events that the job should pause itself on for debugging purposes.
   string debug_pause_on = 20;
+
+  // RestoreSystemUsers set to true if user runs RESTORE SYSTEM USERS.
+  // TODO (msbutler): deprecate after 22.2 branch cut
```

Comment:
nit: is set to true

Also "Delete in 23.1" instead of deprecate maybe?

Reply:
Done.